src/lib/nexus-schema-stateful/stateful-nexus-schema.ts

```diff
@@ -28,6 +29,49 @@ export interface NexusSchemaStatefulBuilders {
   idArg: typeof NexusSchema.idArg
   extendType: typeof NexusSchema.extendType
   extendInputType: typeof NexusSchema.extendInputType
+  /**
+   * Add a GraphQL.js type in your Nexus schema
+   */
+  importType: {
+    /**
+     * Add a GraphQL.js scalar type and (optionally) expose it as a method in your definition builders.
+     * Check the second overload if you're not adding a scalar type.
+     * 
+     * @example
+     * 
+     * ```ts
+     * import { schema } from 'nexus'
+     * import { GraphQLDate } from 'graphql-iso-date'
+     *
+     * schema.importType(GraphQLDate, 'date')
+     *
+     * schema.objectType({
+     *  name: 'SomeObject',
+     *  definition(t) {
+     *   t.date('createdAt') // t.date() is now available (with types!) thanks to `importType`
+     *  },
+     * })
+     * ```
+     *
+     */
+    (scalarType: GraphQL.GraphQLScalarType, methodName?: string): GraphQL.GraphQLScalarType
```

Comment:
Will VSCode display this doc with the little overload arrow nav? I ask because sometimes it does not, but I don't know the rules.

Is the order of nav entries determined by the order in source code here? I ask because you mention here "Check the second overload". Great, so long as that'll be the first one they always see.

Great work!

Reply:
> Will VSCode display this doc with the little overload arrow nav? I ask because sometimes it does not, but I don't know the rules.

Yes, I checked and both docs are displayed depending on which overload you choose.
The overload are also displayed in the order in which they're displayed, hence why I added this note